devsetup/scripts/tripleo.sh

```diff
@@ -97,6 +99,9 @@ export OCTAVIA_ENABLED=${OCTAVIA_ENABLED}
 export TELEMETRY_ENABLED=${TELEMETRY_ENABLED:-true}
 export TLSE_ENABLED=${TLSE_ENABLED:-false}
 export CLOUD_DOMAIN=${CLOUD_DOMAIN:-localdomain}
+export TRIPLEO_NETWORKING=${TRIPLEO_NETWORKING:-true}
+export TRIPLEO_ATTACH_EXTNET=${TRIPLEO_ATTACH_EXTNET:-true}
+export TRIPLEO_CONFIGURE_DEFAULT_ROUTE=${TRIPLEO_CONFIGURE_DEFAULT_ROUTE:-true}
```

Comment:
IIUC the defaults from here are dead code as the variables will always be expanded to whathever was set in the outermost script at the beginning of the file (lines 23-40). The defaults here never come into play. I can see that's already the case for the other vars too so we're just following the pattern here.

So nothing to change in this PR, just noting it as something we should probably refactor separately.